$NetBSD: patch-ac,v 1.3 2011/08/05 18:21:13 wiz Exp $

Be nice and include sys/types.h before sys/ipc.h.

Fix build with latest XShm from xsrc. Hopefully still works with other Xs.

--- libvideogfx/x11/dispimg.cc.orig	2002-05-27 14:02:43.000000000 +0000
+++ libvideogfx/x11/dispimg.cc
@@ -19,6 +19,7 @@
 
 #include "config.h"
 
+#include <sys/types.h>
 #include <assert.h>
 #include <time.h>
 #include <sys/ipc.h>
@@ -32,6 +33,8 @@ using namespace std;
 
 namespace X11 {
 #include <X11/Xlib.h>
+#include <X11/Xmd.h>
+#include <X11/extensions/shmproto.h>
 #include <X11/extensions/XShm.h>
 #include <X11/Xutil.h>
 
